notebooks/09_Monitoring/09_02_Iris_Pipeline_Metrics.ipynb

```diff
@@ -0,0 +1,138 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Iris Pipeline Metrics Visualization Example"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this example notebook, we utilize iris dataset, trained with `sklearn` models, and evaluate with `accuracy` and `roc auc score` metrics. We can visualize the metrics in Pipeline Run page."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Build Pipeline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "1. Run the following command to load Kubeflow Pipelines SDK"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import kfp\n",
+    "from kfp import components\n",
+    "from kfp import dsl"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "2. Build Kubeflow Pipeline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "@dsl.pipeline(\n",
+    "    name='IRIS Classification pipeline metrics',\n",
+    "    description='IRIS Classification using LR in SKLEARN to visualize metrics'\n",
+    ")\n",
+    "def iris_pipeline():\n",
+    "    op = dsl.ContainerOp(name='metrics_vis',\n",
+    "                         image='chuckshow/iris-pipeline-metrics:latest',\n",
```

Comment:
Let's add a section before this. We should tell user, if they like to build their own image, do something. etc

Reply:
Fixed.